activerecord/lib/active_record/persistence.rb

```diff
@@ -173,8 +173,8 @@ def insert!(attributes, returning: nil)
       #     { title: 'Eloquent Ruby', author: 'Russ' }
       #   ])
       #
-      #   # raises ActiveRecord::RecordNotUnique because 'Eloquent Ruby'
-      #   # does not have a unique ID
+      #   # Raises ActiveRecord::RecordNotUnique because 'Eloquent Ruby'
+      #   # does not have a unique id
```

Comment:
We can retain the `unique id` change. We use `unique ID` at other places.

Reply:
Makes sense. Retained back to the original.